Composer/packages/client/src/pages/language-generation/LGPage.tsx

```diff
@@ -25,16 +25,15 @@ const LGPage: React.FC<RouteComponentProps<{
   lgFileId: string;
 }>> = (props) => {
   const { dialogId = '', projectId = '', skillId, lgFileId = '' } = props;
-  const dialogs = useRecoilValue(validateDialogsSelectorFamily(skillId ?? projectId ?? ''));
-
+  const dialogs = useRecoilValue(dialogIdsState(skillId ?? projectId ?? ''));
```

Comment:
Is there any reason why projectId passed to table-view would be emoty string?

skillId ?? projectId would do right?